pages/common/crontab.md

```diff
@@ -13,3 +13,19 @@
 - Remove all cron jobs for the current user:
 
 `crontab -r`
+
+- Cron job definition format:
+
+`{{minute}} {{hour}} {{day_of_month}} {{month}} {{day_of_week}} {{command_to_execute}}`
+
+- Sample job which runs at 10:00 every day. * means any value:
+
+`0 10 * * * {{path/to/script.sh}}`
+
+- Sample job which runs every minute on the 3rd of April:
+
+`* * 3 4 * {{path/to/script.sh}}`
+
+- Sample job which runs at 02:30 every friday:
```

Comment:
I wasn't sure we should outright omit that numbers can be used, since that's a more convenient format, but I guess it's a good thing if we encourage crontabs out there to start becoming more legible xD I view that in the same light as our chmod page not mentioning octal markup, so there's precedent :) go ahead and remove the numbers and ranges where possible.